ledger/participant-integration-api/src/test/suite/scala/platform/apiserver/meteringreport/MeteringReportSpec.scala

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.apiserver.meteringreport
+
+import com.daml.lf.data.Ref
+import com.daml.lf.data.Time.Timestamp
+import com.daml.platform.apiserver.meteringreport.MeteringReport._
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AsyncWordSpec
+import java.time.Duration
+import java.time.temporal.ChronoUnit
+import spray.json.enrichAny
+import spray.json._
+
+class MeteringReportSpec extends AsyncWordSpec with Matchers {
+
+  "the metering report" should {
+
+    "serialize and deserialize as JSON" in {
+
+      val application = Ref.ApplicationId.assertFromString("a0")
+      val from = Timestamp.now()
+      val to = from.add(Duration.of(-1, ChronoUnit.DAYS))
```

Comment:
I don't get it, is `to` 1 day before `from`?

Reply:
Changed to +1